types/testingutils/keygen_protocol.go

```diff
@@ -0,0 +1,15 @@
+package testingutils
```

Comment:
that should be in the dkg package.. testing utils is intended to configure existing structs. We better not use mocks

Reply:
If we want to include the ecdsa signature in spectest, we need mocks so that the messages are deterministic. Otherwise, we have to exclude signature verification in spectest.

Reply:
we can do signature verification instead of signature comparison.